README.md

```diff
@@ -31,6 +31,14 @@ confluent-kafka-dotnet is derived from Andreas Heider's [rdkafka-dotnet](https:/
 We're fans of his work and were very happy to have been able to leverage rdkafka-dotnet as the basis of this
 client. Thanks Andreas!
 
+## Documentation
+
+* [Github Wiki](https://github.com/confluentinc/confluent-kafka-dotnet/wiki)
+* [Blog Posts and Presentations](https://github.com/confluentinc/confluent-kafka-dotnet/wiki/Presentations-&-Blogs)
+* [Hosted by Confluent](https://docs.confluent.io/current/clients/dotnet.html)
```

Comment:
Unclear. How about "Official (and usually outdated) .NET client documentation"

Reply:
i love the casual nature of that suggestion, sold.